job.go

```diff
@@ -79,6 +79,8 @@ type Job struct {
 	Status  StatusCode `json:"status"`
 	Latency string     `json:"latency"`
 	Error   string     `json:"error"`
+	PrevRun time.Time
+	NextRun time.Time
```

Comment:
Discard these new fields on JSON response to avoid duplication with existing `/api/jobs` response, there will be duplicate `prev_run` and `next_run` there.
You could try run the `example` locally to see it yourself.

```suggestion
	PrevRun time.Time `json:"-"`
	NextRun time.Time `json:"-"`
```